resources/hpxml-measures/HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -142,7 +142,9 @@ def self.apply_header(hpxml_header, epw_file)
     end
 
     if hpxml_header.temperature_capacitance_multiplier.nil?
-      hpxml_header.temperature_capacitance_multiplier = 1.0
+      # Different than HPXML defaults, based on work from Jeff Maguire and Yueyue
+      # Discussion located in issue 1161 and will likely need to get updated later.
+      hpxml_header.temperature_capacitance_multiplier = 7.0
```

Comment:
FYI there is a BuildResHPXML measure argument for this: https://github.com/NREL/OpenStudio-HPXML/blob/master/BuildResidentialHPXML/measure.rb#L114-L117.

We could either control it using the yml file (requires bsb updates) or using the options lookup (like `simulation_control_daylight_saving_enabled`).

Reply:
@joseph-robertson : Thanks, I like the options_lookup suggestion. Are you able to revert this and then stub it out in options_lookup. This way it will be easy to change the value and when we all decide as a group what value to use, then we can switch it back to auto.